eclasses/settings.py

```diff
@@ -56,7 +56,11 @@
 
 AUTH_USER_MODEL = 'users.User'
 
-SUPPORTED_SUBMIT_METHODS = ['get', 'post', 'put', 'delete']
+REST_FRAMEWORK = {'DEFAULT_SCHEMA_CLASS': 'rest_framework.schemas.coreapi.AutoSchema'}
```

Comment:
Please group your settings because it helps find and understand what happened in your settings.
For example:
/# AUTH 
AUTH_USER_MODEL = 'users.User'
REST_AUTH_SERIALIZERS = {
    'USER_DETAILS_SERIALIZER': 'users.serializers.UserDetailsSerializer'
}